parts/appendices/D.fodt

```diff
@@ -4225,15 +4225,15 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
      </text:list-item>
     </text:list>
     <text:p text:style-name="_40_TextBody">Some of the C++ code in OPM Flow has been “wrapped” in Python which means that one can invoke the <text:s/>simulator’s C++ code from one’s own Python programs. At the moment the wrappers for the input layer and the code for working with the result files is quite complete and usable. Work is also underway to be able to run the simulator and to interact with running simulations from within Python.</text:p>
-    <text:p text:style-name="_40_TextBody">The goal is for the Python code to be structured like a native Python API, but some design decisions are certainly influenced by the underlying C++ implementation, and to get a deeper understanding of what is possible and how to achieve specialized tasks one might need to consult the simulator’s C++ code.</text:p>
+    <text:p text:style-name="_40_TextBody">The goal is for the Python code to be structured like a native Python <text:a xlink:href="#__RefHeading___Toc4422_421927891">API</text:a>, but some design decisions are certainly influenced by the underlying C++ implementation, and to get a deeper understanding of what is possible and how to achieve specialized tasks one might need to consult the simulator’s C++ code.</text:p>
```

Comment:
API is not used as a keyword in this appendix.  Maybe prevent the link here and below?

Reply:
Yes good idea. I have added a commit to address this.